### DISCUSSION TITLE
header anchor custom id or name

### DISCUSSION THREAD

**Discussion by leosin** [vuejs/vitepress#1563] [Q&A] [answered]
Hi my heros...

i want custom the header anchor

```md
## The Yellow Banana
### I love You
```
i want anchor is custom to `banana` and `love`, not `the-yellow-banana` `i-love-you`

is there any good way?

**Comment by brc-dd** [comment, marked as answer]
```md
## The Yellow Banana {#banana}
```

**Comment by leosin** [comment]
Great, it really worked。
thank you very much! ❤️
